social.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Media Strategy
+tagline: Jupyter Media Strategy
+permalink: /social
+---
+
+The Jupyter Media Strategy Working Group (JMS) exists to ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter.  Please see our [charter](https://jupyter.org/governance/charters/MediaStrategyCharter.html) as and [current list of constituents](https://jupyter.org/governance/people.html#jupyter-media-strategy-working-group) for more information.
+
+## Jupyter Blog
+The Jupyter [blog](https://blog.jupyter.org) is designed for community members to share valuable, educational content, including news and tutorials, about Jupyter.  Please see the Jupyter Blog Submission [Guidelines](/blog_guidelines) for details on creating a post.
+
+## Jupyter Discourse Forum 
+The Discourse [forum](https://discourse.jupyter.org) is meant to be a welcoming place for discussion about Project Jupyter.  Members are encouraged to ask and answer questions, and highlight wins. If you are interested to be a part of the Jupyter Discourse moderation team, please reach out to us at jupyter-media-strategy@googlegroups.com.
+
+## Social Media Accounts
+The JMS maintains a list of official Project Jupyter social media accounts to support and promote our organization's interests.  We primarily use our Mastodon account for interactive social media engagement, but will also broadcast messages about the blog or Jupyter events on other channels. 
+
+If you are organizing a Jupyter event or have something Jupyter-related that should be boosted by the official Jupyter account, please reach out to us at jupyter-media-strategy@googlegroups.com.
+
+List of channels currently maintained by JMS in alphabetical order:
```

Comment:
I woudl add the "active" word in this sentence.

Reply:
Done